<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QRadioButton" name="RBMan">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Man</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RBWoman">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Woman</string>
   </property>
  </widget>
  <widget class="QLabel" name="LSex">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QLabel" name="LStatus">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RBGood">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Good</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RBNormal">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RBAwful">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>110</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Awful</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="RB0019">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>0-19</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB2039">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>20-39</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB4059">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>40-59</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB60">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;60</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PBShow">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>660</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
